### DISCUSSION TITLE
Skip text within brackets

### DISCUSSION THREAD

**Discussion by einaren** [textlint/textlint#888] [Q&A]
Is there a way to skip everything within brackets like <>, [] nd {}. Just like everything within `` seams to be skipped. This would make it far easier to add terms to textlint-rule-terminology (regexp is an option, but I personally gave up..)

**Comment by azu** [comment]
[textlint/textlint-filter-rule-allowlist](https://github.com/textlint/textlint-filter-rule-allowlist) will help you.
This filter rule ignore texts that is matched regexp.

```json
{
    "filters": {
        "allowlist": {
            "allow": [
                "/<.*?>/",
                "/\[.*?]/"
            ]
        }
    }
}
```

- [Ignoring Text · textlint](https://textlint.github.io/docs/ignore.html)

**Reply by einaren** [reply]
Great! Thank you :)
